fava/static/javascript/entry-forms/Posting.svelte

```diff
@@ -39,20 +39,26 @@
   }
 </style>
 
-<div
-  class="fieldset posting"
-  class:drag
-  draggable="true"
-  on:dragstart={dragstart}
-  on:dragenter={dragenter}
-  on:dragover={dragenter}
-  on:dragleave={dragleave}
-  on:drop|preventDefault={drop}>
+<div class="fieldset posting" class:drag>
+  <button
+    class="muted round drag-fieldset"
+    draggable="true"
+    on:dragstart={dragstart}
+    on:dragenter={dragenter}
```

Comment:
This and all the following handlers should still apply to the whole `div`

Reply:
Fixed.